CDF Outreach Committee Charter.md

```diff
@@ -0,0 +1,115 @@
+
+# CDF Outreach Committee Charter
+
+
+## Mission Statement
+
+The Outreach Committee is the place where the community comes together, including the projects, the members, and the ambassadors with the goal to identify opportunities to help market and communicate CDF's projects, SIGs, working groups, etc messages (new releases, need for contributors, etc.) within the CDF community and to the broader technical community in general. The Outreach Chair acts as the communication link between the board and the rest of the community.
+
+
+## Members
+
+Members of the Outreach Committee should consist of one representative from each member company at the Premier, General  and End User membership levels as well as one representative for all adopted CDF projects and the Ambassador chair. This will make up the voting board to be led with direction from the Outreach Committee Chair. Additional project and community members may attend Outreach Committee meetings in a non-voting capacity. Members will be expected to contribute to initiatives aimed at growing the CDF community and supporting CDF projects.
+
+**Voting Members**
+
+
+
+* Member Company Representatives
+* Project Representatives
+* Ambassador Chair
+
+**Non Voting**
+
+
+
+* Observers
+* Governing Board
+* Ambassadors
+
+
+## Communication
+
+Updates will be provided bi-monthly to the community with more formal updates provided monthly to the Governing Board. Upcoming events and initiatives supported by the CDF Outreach Committee will be listed on the community calendar.
+
+Questions? Requests? Comments?....Let us know! 
+
+[Email Outreach Committee Staff](outreach@cd.foundation)
+
+
+## Meetings
+
+
+### **Agendas**
+
+Requests to be added to Outreach Committee agendas should be submitted at least one week prior to the regularly scheduled Outreach Committee meeting. 
```

Comment:
TBH I do not see why the process should be so rigid. Why not just a collaborative Google doc with change suggestions

Reply:
@oleg-nenashev the plan is for these meetings to be interactive and productive which requires planning. Our previous model of running through "updates" was deemed ineffective by the participants so we are working on a new structure.  Considering we will be working closely with the projects, ambassadors, and TOC Chair it should not be difficult to provide agenda items in advance.